File: 백준/Bronze/2753. 윤년/윤년.cc
#include <stdio.h>

int main(void)
{
	//윤년:(연도 % 4 ==0) && 연도 % 100 !=0 || 연도 % 400 == 0

	int year;
	scanf("%d", &year);

	int leap_year = (year % 4 == 0 && year % 100 != 0) || (year % 400 == 0);

	if (leap_year == 1) printf("%d\n", leap_year);
	else printf("%d\n", leap_year);

	return 0;
}

File: 백준/Bronze/2753. 윤년/윤년.py
# 윤년 : 연도가 4의 배수이면서, 100의 배수가 아닐 때 "또는" 400의 배수일 때
year = int(input())

A = year % 4 == 0 # 1999 % 4 = 3 -> True / 반환값을 1 또는 0 으로 해야됨
B = year % 100 != 0
C = year % 400 == 0
yoon = A and B or C

if yoon == True:
    print(1)
else:
    print(0)